src/Wrapper/SeparatedVariation.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\PlentyMarketsRestExporter\Wrapper;
+
+use FINDOLOGIC\PlentyMarketsRestExporter\RegistryService;
+use FINDOLOGIC\PlentyMarketsRestExporter\Response\Entity\Pim\Property\ImageAttributeValue;
+use FINDOLOGIC\PlentyMarketsRestExporter\Response\Entity\Pim\Variation;
+
+class SeparatedVariation
+{
+    private Variation $variation;
+
+    private RegistryService $registryService;
+
+    public function __construct(
+        Variation $variation,
+        RegistryService $registryService
+    ) {
+        $this->variation = $variation;
+        $this->registryService = $registryService;
+    }
+
+    /**
+     * @param ImageAttributeValue[] $imageAttributeValues
+     * @param array $variationAttributes
+     * @return bool
+     */
+    public function isImageAvailable(array $imageAttributeValues, array $variationAttributes): bool
+    {
+        $imageAvailable = false;
+
+        foreach ($imageAttributeValues as $imageAttributeValue) {
+            foreach ($variationAttributes as $variationAttribute) {
+                $attributeData = explode('_', $variationAttribute);
+                $imageAttributeId = $imageAttributeValue->getAttributeId();
+                $imageValueId = $imageAttributeValue->getValueId();
+                $variationAttributeId = $attributeData[0];
+                $variationValueId = $attributeData[1];
+
+                if ($imageAttributeId != $variationAttributeId || $imageValueId != $variationValueId) {
```

Comment:
Why aren't you using a strict comparison?

```suggestion
                if ($imageAttributeId !== $variationAttributeId || $imageValueId !== $variationValueId) {
```